Base/Logger/Logger.Static/FileLogger.hpp

```diff
@@ -2,22 +2,16 @@
 
 #include <Logger/ILogger.hpp>
 #include <Utility/Utility.hpp>
-#include <algorithm>
-#include <chrono>
-#include <condition_variable>
```

Comment:
As I've already stated - this header actually uses std::comdition_variable hence the CI issues. I won't check other headers since I believe you can easily do this using the CI.